### ISSUE TITLE
Cannot edit the CBZ metadata in Ubuntu

### ISSUE THREAD

**Issue by winstonma** [darodi/kcc#40]
I opened KCC v5.6.1 in Ubuntu 22.04.1 and I tried to edit the CBZ metadata by pressing the **Editor** button (Please check the screenshot below).

![image](https://user-images.githubusercontent.com/1215090/214439668-3d3b61cd-a372-4ef4-8cc7-f905dcad93bc.png)

The I got the following error on the console. And the application closed.

```
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Yaru/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Yaru/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
```

Are there any way I could edit metadata in KCC? Thanks

**Comment by winstonma** [comment]
Just checked the system and the file `image-missing.png` is there

```
$ ls /usr/share/icons/Yaru/16x16/status/image-missing.png
/usr/share/icons/Yaru/16x16/status/image-missing.png
```

**Comment by darodi** [comment]
@winstonma 

I've corrected the AppImage
https://github.com/darodi/kcc/releases/tag/v5.6.2b1
is it working now?

flatpak is also available here:
https://flathub.org/apps/details/io.github.ciromattia.kcc

**Comment by winstonma** [comment]
Yes now it's working

Just want to know if I am running correctly on Ubuntu using the following command:
```
GDK_BACKEND=x11 LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6 ./kindleComicConverter-latest-x86_64.AppImage
```

**Comment by darodi** [comment]
@winstonma 

I've tried without `GDK_BACKEND=x11 LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6` and it seemed ok for me.

**Comment by winstonma** [comment]
For the v5.6.2b1 it works fine. But on v5.6.1 I got the following error

```
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.

(kcc_linux_5.6.1:32505): GLib-GIO-ERROR **: 08:08:53.491: Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing'
Trace/breakpoint trap (core dumped)
```

That I have to put `GDK_BACKEND=x11 LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6` in order to open it. Not sure if there are problems on my Ubuntu settings.

**Comment by darodi** [comment]
@winstonma 

v5.6.1 flatpak or AppImage?

**Comment by winstonma** [comment]
I just downloaded the executable from [the release page](https://github.com/darodi/kcc/releases/tag/v5.6.1). I am not sure it is flatpak or AppImage.

**Comment by darodi** [comment]
v5.6.2b1 works fine. I'm closing the issue.
Will be merged in v5.6.2

**Comment by winstonma** [comment]
I know this is closed but just FYI. The flatpak version of KCC v5.6.1 works. It's just the AppImage version of KCC doesn't work.

Of coz I got no problem on KCC v5.6.2 beta
